pyproject.toml

```
@@ -10,8 +10,8 @@ name = "pyhf"
 dynamic = ["version"]
 description = "pure-Python HistFactory implementation with tensors and autodiff"
 readme = "README.rst"
-license = "Apache-2.0"
-requires-python = ">=3.8"
+license = { text = "Apache-2.0" }  # SPDX short identifier
```

This shouldn't get changed back. This was intentionally changed in PR https://github.com/scikit-hep/pyhf/pull/2623.

```suggestion
license = "Apache-2.0"
```